app/templates/road-sign-concepts/index.hbs

```diff
@@ -1,68 +1,89 @@
-        <div class="au-c-data-table__search road-sign-code-search">
+<SidebarContainer>
+  <:sidebar>
+    <div class="au-o-box">
+      <AuHeading @level="2" @skin="3" class="au-u-margin-bottom">
+        {{t "utility.filters"}}
+      </AuHeading>
+      <form class="au-c-form">
+        <AuFormRow>
+          <AuLabel for="label-filter">
+            {{t "road-sign-concept.crud.label-filter"}}
+          </AuLabel>
           <AuInput
+            id="label-filter"
             value={{this.label}}
-            placeholder={{t "road-sign-concept.crud.label-filter"}}
+            placeholder={{t "road-sign-concept.crud.label-filter-placeholder"}}
             @icon="search"
             @iconAlignment="right"
             {{on "input" (perform this.updateSearchFilterTask "label")}}
           />
-        </div>
-        <div class="au-c-data-table__search road-sign-meaning-search">
+        </AuFormRow>
+        <AuFormRow>
+          <AuLabel for="meaning-filter">
+            {{t "road-sign-concept.crud.meaning-filter"}}
+          </AuLabel>
           <AuInput
+            id="meaning-filter"
             value={{this.meaning}}
-            placeholder={{t "road-sign-concept.crud.meaning-filter"}}
+            placeholder={{t "road-sign-concept.crud.meaning-filter-placeholder"}}
             @icon="search"
             @iconAlignment="right"
             {{on "input" (perform this.updateSearchFilterTask "meaning")}}
           />
-        </div>
-        <label>
-          <span class="au-u-hidden-visually">{{t
-              "road-sign-concept.crud.classification-filter"
-            }}</span>
+        </AuFormRow>
+        <div>
+          <AuLabel for="classification-filter">
+            {{t "road-sign-concept.crud.classification-filter"}}
+          </AuLabel>
           <PowerSelect
             @allowClear={{true}}
             @options={{this.model.classifications}}
             @selected={{this.selectedClassification}}
-            @placeholder={{t "road-sign-concept.crud.classification-filter"}}
+            @placeholder={{t "road-sign-concept.crud.classification-filter-placeholder"}}
             @loadingMessage={{t "utility.loading"}}
             @onChange={{this.updateCategoryFilter}}
-            @triggerClass="classification-select"
+            @triggerId="classification-filter"
             as |classification|
           >
             {{classification.label}}
           </PowerSelect>
-        </label>
-        <LinkTo class="au-c-button" @route="road-sign-concepts.new">
-          {{t "road-sign-concept.crud.new"}}
-        </LinkTo>
-      </Group>
-    </AuToolbar>
-  </:menu>
-  <:rowAction as |roadSignConcept|>
-    <LinkTo
-      class="au-c-link"
-      @route="road-sign-concepts.road-sign-concept"
-      @model={{roadSignConcept.id}}
+        </div>
+        <div class="au-u-text-right">
+          <AuButton
+            @skin="naked"
+            @disabled={{this.canResetFilters}}
```

Comment:
The name is confusing, since you would expect the opposite. We need to give it a better name.